spot_rl_experiments/spot_rl/utils/utils.py

```diff
@@ -167,3 +168,30 @@ def __setitem__(self, key, value):
         if self._maxlen > 0:
             if len(self) > self._maxlen:
                 self.popitem(False)
+
+
+def runner():
+    import yaml
+    import json
+
+    # Read the YAML file
+    with open(WAYPOINTS_YAML, 'rb') as file:
+        data = yaml.safe_load(file)
+
+    # Prepare to store all waypoint data in a single JSON structure
+    all_waypoints = []
+
+    # Iterate over each waypoint and add it to the list
+    for key, values in data['nav_targets'].items():
+        if len(values) == 3:
+            x, y, _ = values
+            all_waypoints.append({key: {"x": x, "y": y}})
+
+    # Write the combined JSON data to a file
+    with open(DATA_WAYPOINTS_JSON, 'w') as file:
+        json.dump(all_waypoints, file, indent=2)
+
+    print("Combined JSON data has been written to 'all_waypoints.json'.")
+
+if __name__ == "__main__":
```

Comment:
This should also be a part of separate PR along with `create_PCD_from_waypoints.py`

Reply:
removed